src/main/java/org/apache/datasketches/req/FloatBuffer.java

```diff
@@ -19,13 +19,12 @@
 
 package org.apache.datasketches.req;
 
```

Comment:
it seems to me that getCountWithCriterion() method is not used anymore. it was called by getRank() which uses sorted view now

Reply:
You are correct.  I will remove it and check the other methods in this class.

Reply:
I changed my mind.  This "getCountWithCriterion()" method is used in a number of places in the ReqFloatBufferTest and removing it would also remove a number of other important tests of the FB.   Being able to test the FloatBuffer separately from the rest of the ReqSketch is valuable as it gives us confidence that the FB is operating correctly.  

The ReqSketch is a pretty complicated sketch so I am reluctant to remove code that could help us debug problems in the future.

Reply:
Keeping this method for now.